geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl

```diff
@@ -25,15 +28,19 @@ default_project_file:
     - \.circleci/config\.yml
     - \.travis\.yml
     - qgisserver/geomapfish\.yaml
-    - build
   exclude:
+    - geoportal/{{package}}_geoportal/static/apihelp
     - geoportal/{{package}}_geoportal/static-ngeo/js/{{package}}module\.js
     - mapserver/demo\.map\.tmpl
 
+# Files that we dont probide any diff => will just be keeped as is.
 no_diff:
   - .*\.po
   - CONST_.+
   - .*/CONST_.+
+
+# Files that will be present in the CONST_create_template but will not be considered in the upgrade.
+# Used to provide the alt application without disturbing the user during upgrade.
```

Comment:
Used to provide the alt application => application**s** without disturbing the user during upgrade.

Reply:
=> Used to provide the alt application => does not disturb the user during upgrade.

Reply:
It was for the 's' because there is more than one application
application => application**s**

Reply:
+1 Thanks